index.html

```diff
@@ -1,4 +1,4 @@
-<!DOCTYPE html>
+﻿<!DOCTYPE html>
```

Comment:
Was there a change you made on line 1?

My GUI is showing a diff, but I can't see what changed.

Reply:
No, there wasn't any change that was made. I edit using Atom and sometimes it reorients the code by shifting the first or last line, so that might be it.

Reply:
If that’s the case can you make another change undoing that line change?

The total changes that your PR should encompass should only be the contributing link.

You can look at using the patch flag when staging your files too. 

Thanks

Reply:
@abhinavmasalia will you be updating your PR?

Reply:
@jwu910 Hello, yes sorry for the delay. Something's broke in my fork and being new to this, I ended up deleting the fork to start anew. I'll be closing this PR and opening a fresh PR with just the required change. Really sorry about that.